.gitignore

```diff
@@ -89,5 +89,4 @@ fastlane/test_output
 
 iOSInjectionProject/
 .DS_Store
-Chrono/Chrono/GoogleService-Info.plist
-Chrono/GoogleService-Info.plist
+GoogleService-Info.plist
```

Comment:
I dont think this is a wild card ignore. i think we want */GoogleService-Info.plist or bring the two lines this deletes over.

Reply:
I was thinking the same thing but I checked it out on git and it is does work 👍🏻